code/modules/mining/fulton.dm

```diff
@@ -35,6 +35,7 @@ var/list/total_extraction_beacons = list()
 		if(!A)
```

Comment:
While you are at this file, can you also change the input two lines above to be `input(..) as null|anything in possible_beacons` so it has a cancel button, please?